Source/WTF/wtf/Assertions.h

```diff
@@ -268,9 +268,15 @@ WTF_EXPORT_PRIVATE bool WTFIsDebuggerAttached(void);
 #define CRASH_GPR5 "r14"
 #define CRASH_GPR6 "r13"
 
-#elif CPU(ARM64) // CPU(X86_64)
+#elif CPU(ARM64)
 
+#if ASAN_ENABLED
+#define CRASH_CODE 0x0
+#define CRASH_INST "brk #0x0"
+#else
+#define CRASH_CODE 0xc471
 #define CRASH_INST "brk #0xc471"
+#endif
```

Comment:
It seems like these could be guarded on the value not being defined already?

Reply:
I think if there's a collision, we want it to error out.  So, I think keeping it as is would work.  Duplicate definitions will result in an error.

Reply:
I think the idea is that some other project wants to set their own `CRASH_CODE`. Right now they can't without modifying the source code. if this was
```c++
#if ASAN_ENABLED
#if !defined(CRASH_CODE)
#define CRASH_CODE 0x0
#end
#define CRASH_INST "brk #0x0"
#else
#if !defined(CRASH_CODE)
#define CRASH_CODE 0x0
#end
#define CRASH_INST "brk #0xc471"
#endif
``` They would be able to.

Reply:
Alternatively, the folks we're making this change for could set those values and we could just do:
```c++
#if !defined(CRASH_CODE)
#define CRASH_CODE 0xc471
#endif

#if !defined(CRASH_INST)
#define CRASH_INST "brk #" STRINGIZE_VALUE_OF(CRASH_CODE)
#endif
```

Reply:
It is important that we use 0xc471 in production.  Hence, I would not want an option to be able to silently replace it with something else.

Reply:
Almost any feature flag can be overridden by setting a value before building causing who knows what problems. I don't think this is much different.

Reply:
Talked with Keith about this offline.  His main concern is to empower custom builds like ASAN to specify whatever crash code they want in case they want some specific one.  I think it is rare that this would be needed, and I fear some innocent code accidentally overriding it silently.  

However, I can accommodate this with reduced risk by renaming CRASH_CODE to something like WTF_FATAL_CRASH_CODE, which is less likely to be defined willy nilly by random code.

Reply:
I think we should err on the side of making ASAN folks change source code rather than risk a random security regression.

Reply:
If we’re going to assume someone can set these values why couldn’t  someone just set ASAN_ENABLED to 1?